src/TwilioVideo.android.js

```diff
@@ -83,8 +83,8 @@ const nativeEvents = {
 };
 
 class CustomTwilioVideoView extends Component {
-  connect({ accessToken }) {
-    this.runCommand(nativeEvents.connectToRoom, [accessToken]);
+  connect(roomName, accessToken) {
```

Comment:
On iOS, I believe the parameters are passed in as an object so we should maintain the same interface here.